packages/static-wado-creator/lib/mkdicomwebConfig.js

```diff
@@ -37,11 +38,6 @@ const { mkdicomwebConfig } = ConfigPoint.register({
         description: "Maximum length of public binary data",
         defaultValue: 128 * 1024 + 2,
       },
-      {
```

Comment:
Moved into the general config values so it can be set correctly (was not working previously)